packages/experimental/Dropdown/src/Option/useOption.ts

```diff
@@ -1,8 +1,8 @@
+import { useAsPressable } from '@fluentui-react-native/interactive-hooks';
 import { OptionProps, OptionState } from './Option.types';
 
-export const useOption = (_props: OptionProps): OptionState => {
-  return {
-    props: {},
-    state: {},
-  };
+export const useOption = (props: OptionProps): OptionState => {
+  const pressable = useAsPressable(props);
+
+  return pressable;
```

Comment:
(Feel free to reject this suggestion)
Could we try using Pressable from `react-native` along with `usePressableState` for new components? Super Ok if you'd rather leave that for later.

Reply:
Is Pressable ready for use?

Reply:
It should be. The only think lacking from when I last worked on it was types in the DefinitelyTyped repo.

Reply:
I think I'll commit this for now so it doesn't get too out of sync from main, and work on the Pressable separately.